tests/tests/parsers/code_22f2.log

```diff
@@ -6,3 +6,6 @@
 # Vasco HRU D60
 2024-10-15T12:32:31.600425 000 RQ --- 18:130140 32:022222 --:------ 22F2 001 00            # {}
 2024-10-15T12:32:31.615141 082 RP --- 32:022222 18:130140 --:------ 22F2 006 000000010000  # [{'hvac_idx': '00', 'measure': 0.00}, {'hvac_idx': '01', 'measure': 0.00}]
+
+# ClimaRad MiniBox
+2024-10-23T11:55:36.231336 077 RP --- 29:099029 18:130140 --:------ 22F2 003 000AAF         # [{'hvac_idx': '00', 'measure': 27.35}]
```

Comment:
Is this data also in a contemporaneous `31DA` / `31D9` packet?

Can you confirm what it is measuring - Do you know what the units are?

Reply:
```
2024-10-18T19:34:40.870199 ... RP --- 29:099029 18:130140 --:------ 22F2 003 000933  # [{'hvac_idx': '00', 'measure': 23.55}]
2024-10-18T19:35:04.659092 ...  I --- 29:099029 --:------ 29:099029 31D9 003 0000C8  # {'hvac_id': '00', 'exhaust_fan_speed': 1.0, 'fan_mode': 'C8', 'passive': False, 'damper_only': False, 'filter_dirty': False, 'frost_cycle': False, 'has_fault': False, '_flags': [0, 0, 0, 0, 0, 0, 0, 0]}
2024-10-18T19:35:10.848593 ... RP --- 29:099029 18:130140 --:------ 22F2 003 000A82  # [{'hvac_idx': '00', 'measure': 26.9}]
```

Reply:
So: 
 - `000001` -> `01` -> 1/200 -> 0.005 -> 0.5% -> trickle?
 - `0000C8` -> `C8` -> 200/200 -> 1.000 -> 100% -> boost?